Update leftover review comments from adding property type constraints (#619)
neo4j/docs-cypher: modules/ROOT/pages/constraints/examples.adoc

[[constraints-examples-node-uniqueness]]
== Node property uniqueness constraints

> A node property uniqueness constraint ensures that certain nodes have a set of certain properties whose combined value is unique when all properties exists on the node.

Reviewer: Double "certain" reads awkwardly. 

```suggestion
A node property uniqueness constraint ensures that certain nodes have a set of specified properties whose combined value is unique when all properties exist on the node.
```

CREATE CONSTRAINT book_title FOR (book:Book) REQUIRE book.title IS UNIQUE
----

In this case, the constraint cannot be created because it is in conflict with the existing graph.

> Either use xref::indexes-for-search-performance.adoc[] instead, or remove or correct the offending nodes and then re-apply the constraint.

Reviewer: ```suggestion
Either use xref::indexes-for-search-performance.adoc[] instead, or remove/correct the offending nodes and then re-apply the constraint.
```

_This feature was introduced in Neo4j 5.7._

> A relationship property uniqueness constraint ensures that certain relationships have a set of certain properties whose combined value is unique when all properties exists on the relationship.

Reviewer: ```suggestion
A relationship property uniqueness constraint ensures that certain relationships have a set of specified properties whose combined value is unique when all properties exist on the relationship.
```

CREATE CONSTRAINT series_title FOR ()-[sequel:SEQUEL_OF]-() REQUIRE (sequel.seriesTitle) IS UNIQUE
----

In this case, the constraint cannot be created because it is in conflict with the existing graph.

> Either use xref::indexes-for-search-performance.adoc[] instead, or remove or correct the offending relationships and then re-apply the constraint.

Reviewer: ```suggestion
Either use xref::indexes-for-search-performance.adoc[] instead, or remove/correct the offending relationships and then re-apply the constraint.
```

[[constraints-examples-node-property-existence]]
== Node property existence constraints

> A node property existence constraint ensures that certain nodes have a certain property.

Reviewer: ```suggestion
A node property existence constraint ensures that certain nodes have a specified property.
```

[[constraints-examples-node-key]]
== Node key constraints

> A node key constraint ensures that certain nodes have a set of certain properties whose combined value is unique and all properties in the set are present.

Reviewer: Do we need to explain "the set"
```suggestion
A node key constraint ensures that certain nodes have a set of specified properties whose combined value is unique and all properties in the set are present.
```

.+CREATE CONSTRAINT+
======

Create a node key constraint on the property `born` on nodes with the `Actor` label, when a node without a `born` property already exists in the graph:

.Query
[source, cypher, role=test-fail]
----
CREATE CONSTRAINT actor_born FOR (actor:Actor) REQUIRE (actor.born) IS NODE KEY
----

In this case, the node key constraint cannot be created because it is in conflict with the existing graph.

> Either use xref::indexes-for-search-performance.adoc[] instead, or remove or correct the offending nodes and then re-apply the constraint.

Reviewer: ```suggestion
Either use xref::indexes-for-search-performance.adoc[] instead, or remove/correct the offending nodes and then re-apply the constraint.
```
